custom_components/example_load_platform/__init__.py

```diff
@@ -9,6 +9,6 @@ def setup(hass, config):
         'temperature': 23
     }
 
-    hass.helpers.discovery.load_platform('sensor', DOMAIN, {}, config)
+    hass.helpers.discovery.load_platform(hass, 'sensor', DOMAIN, {}, config)
```

Comment:
In this case we're using the `hass.helpers`, which will automatically add `hass` as the first argument.